adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBNativeRenderer.m

```diff
@@ -0,0 +1,199 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "GADPangleRTBNativeRenderer.h"
+#import "GADMediationAdapterPangleConstants.h"
+#import <BUAdSDK/BUAdSDK.h>
+#import "GADMAdapterPangleUtils.h"
+#include <stdatomic.h>
+#import "GADPangleNetworkExtras.h"
+
+static NSString *const BUDNativeAdTranslateKey = @"bu_nativeAd";
+
+@interface GADPangleRTBNativeRenderer()<BUNativeAdDelegate> {
```

Comment:
For Android, Pangle does its own show event handling and click event handling.
https://github.com/googleads/googleads-mobile-android-mediation/pull/404/files#diff-87954a0754b43f151efc8daddd998c7625dd934cc03c68c98897a32e07926750R126-R128

Does iOS do the same?
https://developers.google.com/ad-manager/mobile-ads-sdk/ios/api/reference/Protocols/GADMediationNativeAd?hl=en

Reply:
Yes,`-nativeAdDidClick:withView:` & `-nativeAdDidBecomeVisible:`